engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetCameraToReflected.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.rendering.dag.stateChanges;
+
+import java.util.Objects;
+import org.terasology.rendering.cameras.Camera;
+import org.terasology.rendering.dag.RenderPipelineTask;
+import org.terasology.rendering.dag.StateChange;
+import org.terasology.rendering.dag.tasks.SetCameraReflectedModeTask;
+
+/**
+ * Instances of this state change generate the tasks that set or reset the reflected flag of a given camera.
+ */
+public class SetCameraToReflected implements StateChange {
```

Comment:
I did consider ReflectCamera. It does sound nice and concise. I'm only concerned about the "Reflect" term as it could be interpreted as related to java Reflection, while here it's more about mirroring/flipping an image rather than deep-inspecting a java object.

Did you interpret it that way by any chance?